src/libcalamaresui/utils/Paste.cpp

```diff
@@ -9,20 +9,35 @@
 
 #include "Paste.h"
 
+#include "Branding.h"
 #include "utils/Logger.h"
 
 #include <QFile>
 #include <QRegularExpression>
 #include <QTcpSocket>
 #include <QUrl>
+#include <QClipboard>
+#include <QApplication>
+#include <QStringList>
```

Comment:
Canonically, includes are ordered alphabetically within a block; the `calamaresstyle` script will do the right thing (using astyle and clang-format).